Vic2ToHoI4/Source/HOI4World/HoI4Localisation.cpp

```diff
@@ -482,6 +482,60 @@ void HoI4::Localisation::createGeneratedDominionLocalisations(const std::string&
 }
 
 
+void HoI4::Localisation::createUnrecognizedNationLocalisations(const std::string& tag,
+	 const Country& nation,
+	 const Vic2::Localisations& vic2Localisations,
+	 const Mappers::CountryNameMapper& countryNameMapper,
+	 const std::set<std::string>& majorIdeologies,
+	 const ArticleRules& articleRules)
+{
+	const auto region = nation.getRegion();
+
+	const auto unrecognizedNameLocalisations = vic2Localisations.getTextInEachLanguage("unrecognized_name");
+	const auto unrecognizedAdjectiveLocalisations = vic2Localisations.getTextInEachLanguage("unrecognized_adjective");
+	for (const auto& ideology: majorIdeologies)
+	{
+		Vic2::LanguageToLocalisationMap localisationsForGovernment;
+		if (const auto localisations = vic2Localisations.getTextInEachLanguage("unrecognized_" + region);
+			 !localisations.empty())
+		{
+			localisationsForGovernment =
+				 configureDominionNames(localisations, unrecognizedNameLocalisations, unrecognizedAdjectiveLocalisations);
+		}
+		else
+		{
+			for (const auto& [language, localisation]: unrecognizedAdjectiveLocalisations)
+			{
+				auto newLocalisation = localisation;
+				if (const auto& secondPart = vic2Localisations.getTextInLanguage(region, language); secondPart)
+				{
+					newLocalisation += " " + *secondPart;
```

Comment:
I'm unsure what you're adding to what here, can't follow the code that deep. Just rechecking if word order is correct for non-english locs where $overlord_ADJ$ goes after territory name. Maybe it's unrelated to this code but I couldn't find tests relating to this.

Reply:
Added comments for clarity. Unfortunately HoI4 world is generally missing tests.